Duplicates.txt

```diff
@@ -1,154 +0,0 @@
-[
```

Comment:
why is it here?

Reply:
This file was supposed to be deleted but somehow it was in the project.